include/FoamAdapter/readers/foamDictionary.hpp

```diff
@@ -32,17 +32,19 @@ bool checkEntryType(const Foam::entry& entry)
     }
     catch (const Foam::IOerror& ioErr)
     {
+        FatalError.throwExceptions(false);
```

Comment:
maybe add a docstring/comment what is happening here? Also i think this can be refactored by having a return variable.